apps/wallet/tests/fixtures.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { test as base, chromium, type BrowserContext } from '@playwright/test';
+import path from 'path';
+
+const EXTENSION_PATH = path.join(__dirname, '../dist');
+const LAUNCH_ARGS = [
+    `--disable-extensions-except=${EXTENSION_PATH}`,
+    `--load-extension=${EXTENSION_PATH}`,
+    // Ensure userAgent is correctly set in serviceworker:
+    '--user-agent=Playwright',
+];
+
+export const test = base.extend<{
+    context: BrowserContext;
+    extensionUrl: string;
+}>({
+    // eslint-disable-next-line no-empty-pattern
+    context: async ({}, use) => {
```

Comment:
Do we have to destruct the first parameter for some reason? Why not just use `_` since we don't use this param.

Reply:
Great question, Playwright yells at me if I don't destructure the first parameter into nothing, seems like they have some hard requirement on it.